packages/code-du-travail-frontend/src/conventions/Convention/__tests__/__snapshots__/index.test.js.snap

```diff
@@ -978,9 +976,7 @@ exports[`<Convention /> renders 1`] = `
               class="accordion"
               data-accordion-component="Accordion"
             >
-              <div
-                id="jours-feries-et-ponts-dans-le-secteur-prive"
```

Comment:
this break the anchor linking ? like page.html#jours-feries-et-ponts-dans-le-secteur-prive

Reply:
I removed the ID here because we don't link to sections in conventions, and there is no way to get this link, not a problem in my opinion

Reply:
I also removed the others because they were badly formatted (spaces in the ID, which is not recommended)

Reply:
d'acc ! dans l'idée j'avais ouvert #2940 car ca me semble être intéressant comme besoin (idéalement avec un mécanisme qui permet de récupérer l'ancre facilement. mais on verra comment on priorise ça

Reply:
Ah oui j'avais pas vu, j'aime beaucoup l'idée !